bin/hicAggregateContacts

```diff
@@ -288,14 +288,15 @@ def main(args):
             ax.set_zticklabels([])
             if vmax is not None and vmax is not None:
                 ax.set_zlim(vmin, vmax)
-
+                
+        if args.outFilePrefixMatrix:
+            output_matrix_name=("".join((args.outFilePrefixMatrix.name, "_", chrom, ".tab")))
```

Comment:
I suggest to write this line as:
"{}_{}.tab".format(args.outFilePrefixMatrix.name, chrom)

or more verbose:
"{file}_{chrom}.tab".format(file=args.outFilePrefixMatrix.name, chrom=chrom)